deploy-edpm-reuse.yaml

```diff
@@ -2,6 +2,15 @@
 - name: Manage unique ID
   ansible.builtin.import_playbook: playbooks/unique-id.yml
 
+- name: Reboot controller-0 to make sure there are no running deployments
+  hosts: controller-0
+  gather_facts: false
+  tasks:
+    - name: Reboot controller-0
+      ansible.builtin.reboot:
+        post_reboot_delay: 600
```

Comment:
Thanks @eshulman2 for incorporating my feedback. `post_reboot_delay` is a blocking activity, which will stop test execution for 10 minutes. Instead, you could use features from both `reboot` and `wait_for_connection`.
`reboot` can use `post_delay` while `wait_for_connection` will continue the process immediately

```
    ansible.builtin.reboot:
        reboot_timeout: 600
        post_reboot_delay: 60
      become: true

    - name: Wait for controller-0 to come back online
      ansible.builtin.wait_for_connection:
        timeout: 600
```

Reply:
The delay might be adjusted